bundles/EcommerceFrameworkBundle/Model/AbstractOrderItem.php

```diff
@@ -129,8 +129,7 @@ public function setTotalPrice($totalPrice)
      */
     public function getTotalNetPrice()
     {
-        //prevent throwing an exception for backward compatibility
-        Logger::err('getTotalNetPrice not implemented for ' . get_class($this));
+        throw new UnsupportedException('getTotalNetPrice is not implemented for ' . get_class($this));
```

Comment:
This changes are BC breaks, we should put a `@TODO` here for v7.

Reply:
I add it to the Upgrade Notes. It is not possible for v6.5.0?

Reply:
Better not, I'm tired of those discussions 😉

Reply:
;)

Reply:
Okay 😄 I will revert this.
How should we fix the error here? Should I return 0? Or should we stay this error till v7?

Reply:
What about 
```php
    /**
     * @return void
     */
    public function getTaxInfo()
    {
        //prevent throwing an exception for backward compatibility
        Logger::err('getTaxInfo not implemented for ' . get_class($this));
    }
```

Reply:
Yes is also possible. I just thought it would be good for developers to know what the function should return. Should then be commented about `@return void`

For v7 it would be good to convert it in a abstract class with abstract methods 😉

Reply:
@brusch I changed it

Reply:
@brusch Now we have new errors like
`Result of method Pimcore\Bundle\EcommerceFrameworkBundle\Model\AbstractOrder::getTaxInfo() (void) is used.` 😄

Reply:
Oh dear 😁 Maybe we should just exclude them as they get changed anyway with v7.

Reply:
Yes we should fix so much as possible and the rest ignore via the baseline feature (https://github.com/pimcore/pimcore/issues/5725)